pkg/agent/controller/agent.go

```diff
@@ -89,10 +85,18 @@ func NewWithDetail(spec *AgentSpecification, syncerConf *broker.SyncerConfig, re
 		BrokerResourcesEquivalent: agentController.serviceImportEquivalent,
 	}
 
+	endpointSliceSyncer := broker.ResourceConfig{
+		LocalSourceNamespace:      metav1.NamespaceAll,
+		LocalResourceType:         &discovery.EndpointSlice{},
+		BrokerResourceType:        &discovery.EndpointSlice{},
+		BrokerResourcesEquivalent: agentController.endpointSliceEquivalent,
+	}
```

Comment:
Won't this sync `EndpointSlices` from all services? Don't we want to only sync ones for exported services that we create?

Reply:
=changed to use admiral API.